File: dayFour/java/afterBreak/QuestionFour.java
import java.util.Scanner;
public class QuestionOne{
    public static void main(String [] args){
    Scanner input = new Scanner(System.in);
    
     int sum = 0;
     
    for(int count = 1; count <+ 10; count++){
        System.out.print("Enter score: ");
        int score = input.nextInt();
        if(count % 2 == 0){
             sum += score;
        }
 
        }
        
    System.out.println("Sum: "+ sum);
    
    }
}

File: dayFour/javaScript/afterBreak/questionFour.js
let sum = 0;
for(let count = 1; count <= 10; count++){
    const scanner = require("prompt-sync")();
    let score = Number(scanner ("Enter score: "));
    if(count % 2 === 0){      
    sum += score;
           }
        }
        
   console.log(sum);
    





